benzina/src/typed_uuid.rs

```diff
@@ -88,6 +92,13 @@ macro_rules! typed_uuid {
             $vis struct $name(::uuid::Uuid);
 
             impl $name {
+                /// Creates a new, unchecked typed Uuid
+                #[must_use]
+                #[cfg(feature = "test")]
+                pub fn new_unchecked(inner: ::uuid::Uuid) -> Self {
```

Comment:
`_unchecked` sounds like something that has to do with unsafe.

Reply:
@paolobarbolini Just suggested me a convention: using the `dangerous_` prefix, which is used [in Rustls](https://docs.rs/rustls/latest/rustls/enum.Connection.html#method.dangerous_extract_secrets) :tada: